soda_machine.rb

```diff
@@ -7,12 +8,18 @@ def initialize(args = {})
   end
 
   def current_inventory_count
+  args[:sodas].length
   end
 
   def find_soda(soda_brand)
+    args[:sodas].include?(soda_brand)
+    return soda_brand
   end
```

Comment:
You have a similar issue here as above.  But you're also looking for a string (e.g. "Pepsi") in an array of `Soda` objects.  You definitely won't find the string, and you're always returning the same thing (`soda_brand`) anyway so that line's not getting you anywhere.